environment.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
     - albumentations==0.4.3
     - basicsr>=1.3.4.0
     - diffusers==0.3.0
-    - einops==0.3.0
+    - einops>=0.3.0
```

Comment:
Why was this changed? If there's a new version and it does not cause issues, specify the new version, otherwise versions should be pinned to avoid breaking changes.

Reply:
Tensorflow needs einops>=0.3.0 otherwise, "AttributeError: module 'keras.backend' has no attribute 'is_tensor'" is thrown when loading keras model. Tensorflow is for depth from image estimation, used for automatic mask selection in scn2img.

I changed it to pin the versions.